sdk/identity/identity/test/public/node/authorizationCodeCredential.spec.ts

```diff
@@ -2,67 +2,98 @@
 // Licensed under the MIT license.
 
 import assert from "assert";
```

Comment:
debt: we should be using `chai` assertions instead of the assert module

Reply:
I’ll remove this correlationId code. It comes from v2. I’ll address this in v2: https://github.com/Azure/azure-sdk-for-js/issues/16436

Reply:
Identity v1 doesn’t have chai as a dependency, but v2 won’t use assert 👍 I can promise that.